content/ja/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 設定ファイルを使用してSecretを管理する
+content_type: task
+weight: 20
+description: リソース設定ファイルを使用してSecretを作成する
+---
+
+<!-- overview -->
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+<!-- steps -->
+
+## 設定ファイルを作成する
+
+先にJSONやYAML形式のファイルでSecretを作成してから、そのオブジェクトを作成することができます。
+[Secret](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#secret-v1-core)リソースには、`data`と`stringData`の2つのマップが含まれています。
+`data`フィールドは任意のデータを格納するのに使用され、base64でエンコードされます。
+`stringData`フィールドは利便性のために用意されており、Secretデータをエンコードされていない文字列として提供することができます。
+`data`と`stringData`のキーは、英数字、`-`、`_`、`.`で構成されている必要があります。
+
+たとえば、`data`フィールドを使用して2つの文字列をSecretに格納するには、次のように文字列をbase64に変換します:
+
+```shell
+echo -n 'admin' | base64
+```
+
+出力は次のようになります:
+
+```
+YWRtaW4=
+```
+
+```shell
+echo -n '1f2d1e2e67df' | base64
+```
+
+出力は次のようになります:
+
+```
+MWYyZDFlMmU2N2Rm
+```
+
+以下のようなSecret設定ファイルを記述します:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: MWYyZDFlMmU2N2Rm
+```
+
+なお、Secretオブジェクトの名前は、有効な[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)である必要があります。
+
+{{< note >}}
+SecretデータのシリアライズされたJSONおよびYAMLの値は、base64文字列としてエンコードされます。
+これらの文字列の中では、改行は有効ではなく、省略する必要があります。
```

Comment:
in https://kubernetes.io/ja/docs/concepts/configuration/secret/ , written as:

```
シリアライズされたJSONやYAMLの機密データはBase64エンコードされています。 
文字列の中の改行は不正で、含まれていてはなりません。 
```

I myself prefer `Secretデータ` rather than `機密データ` so I love your translation, but I think `不正な` is better than `有効でなく` .  so I suggest:

```
SecretデータのシリアライズされたJSONおよびYAMLの値は、base64文字列としてエンコードされます。
文字列の中の改行は不正で、含まれていてはなりません。
```

How do you thinnk?

Reply:
nit: I'd write `文字列中` instead because の~の sounds a bit lengthy

Reply:
@inductor thank you for your comments.
you mean, you prefer `文字列中の改行は不正で、` instead of `文字列の中の改行は不正で、` , right?
it sounds good :)